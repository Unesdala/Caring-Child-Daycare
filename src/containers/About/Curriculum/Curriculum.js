// @ts-nocheck
import React, { Component } from 'react';
import commonUtils from '../../../lib/commonUtils';
import DefaultCurriculumContent from './CurriculumContent';
import DefaultFooter from '../../../components/Footer/Footer';

export class Curriculum extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Curriculum', window.screen.width); }

  render() {
    return (
      <>
        <div className="page-content">
          <DefaultCurriculumContent />
        </div>
        <DefaultFooter />
      </>
    );
  }
}

export default Curriculum;
