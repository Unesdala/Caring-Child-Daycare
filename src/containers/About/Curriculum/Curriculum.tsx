import { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultCurriculumContent from './CurriculumContent';

export class Curriculum extends Component {
  commonUtils: { setTitleAndScroll: (pageTitle: string, width: number) => void };

  constructor(props: Record<string, unknown>) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Curriculum', window.screen.width); }

  render(): JSX.Element {
    return (
      <>
        <div className="page-content">
          <DefaultCurriculumContent />
        </div>
      </>
    );
  }
}

export default Curriculum;
