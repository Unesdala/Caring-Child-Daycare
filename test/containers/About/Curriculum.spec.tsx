/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Curriculum } from 'src/containers/About/Curriculum/Curriculum';
import CurriculumContent from 'src/containers/About/Curriculum/CurriculumContent';

describe('Curriculum', () => {
  const wrapper = shallow(<Curriculum />);
  it('renders', () => {
    expect(wrapper.find(CurriculumContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(CurriculumContent).dive().find('main.curriculum-main--content').exists()).toBe(true);
  });
  
});
