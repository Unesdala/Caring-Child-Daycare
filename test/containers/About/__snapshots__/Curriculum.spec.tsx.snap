// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Curriculum renders 1`] = `ShallowWrapper {}`;
